node/sync.go

```diff
@@ -166,6 +169,49 @@ func (d *Pegnetd) SyncBlock(ctx context.Context, tx *sql.Tx, height uint32) erro
 		return context.Canceled
 	}
 
+	////////////////////////
+	// Zeroing funds at Global Burn Address
+
+	// One time operation, Inserts negative balance for the burn address that used during the attack
+	// We need to do this before main logic because sqlite db will be locked
+	if d.Sync.Synced+1 == V20HeightActivation {
+
```

Comment:
@sigrlami , it is a new transaction, so this should be saved in Tx chain, I think.

Reply:
@StarNeit added special `InsertZeroingCoinbase` to record